docs/roman/associations/skycell_asn.rst

```diff
@@ -87,4 +109,4 @@ the F158 filter.
 
 .. code-block:: text
 
-		skycell_asn r0099101???003*_*_cal.asdf  -o  r0099101 --product-type pass
+		skycell_asn r0099101???003*_*_cal.asdf  -o  r0099101 --product-type p
```

Comment:
This is making a visit level product, so I think I'd suggest
```suggestion
		skycell_asn r0099101001003001_*_cal.asdf  -o  r00991 --product-type visit
```
?

Reply:
Changed